docs/installation.rst

```diff
@@ -41,3 +41,12 @@ context processor:
         ...
         'django.template.context_processors.request',
     )
+
+
+WSGI + Nginx
+#####
+
+1. Add **SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')** to your settings.py 
```

Comment:
[Code blocks are ideally wrapped in backticks in rST.](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html)